frontend/lib/issues/issue-pages.tsx

```diff
@@ -51,19 +62,90 @@ type IssuesAreaPropsWithCtx = IssuesRouteAreaProps & {
   toHome: string;
 };
 
+/**
+ * Category headings that will appear immediately above
+ * certain issues.
+ */
+const CATEGORY_HEADINGS: Map<IssueChoice, string> = new Map([
+  ["BATHROOMS__MOLD", "General"],
+  ["BATHROOMS__SINK", "Sink"],
+  ["BATHROOMS__TUB", "Bathtub"],
+  ["BATHROOMS__SHOWER_MOLD", "Shower"],
+]);
+
```

Comment:
Ok so let me make sure I got this— given this implementation, we only need to list here issue choices that _begin_ a category section so that we can render the category header right above it. Therefore, we don't need to duplicatively list out each issue check list item with whatever category it is associated with. BUT, this also means that the order of our issue checklist items is very important and must always be ordered with categories of issues grouped together.

Reply:
Yup, that is exactly right!